package.json

```diff
@@ -0,0 +1,54 @@
+{
+  "name": "input-plugin-year",
+  "version": "0.0.1",
+  "description": "A simple select input for years based on a range",
+  "main": "index.js",
+  "scripts": {
+    "test": "npm run nyc",
+    "pretest": "npm run lint",
+    "ava": "ava | tap-diff",
+    "ava:watch": "ava --watch | tap-diff",
+    "nyc": "nyc --all npm run ava",
+    "lint": "eslint index.js lib && eslint -c ./.eslintrc-ava.yml tests",
```

Comment:
It doesn't? I see it here: https://github.com/rachelnicole/input-plugin-year/blob/e6131becb8de621238e51e1ccb8d4dd4fef67270/.eslintrc-ava.yml